docs/tutorials/kubectl/app-management-overview.md

```diff
@@ -0,0 +1,238 @@
+---
+title: App Management Overview
+---
+
+{% capture overview %}
+
+*Kubectl* supports several different approaches for creating and managing
+applications in a Kubernetes cluster.  This document provides an overview
+of the different approaches and which environment each approach is best
+suited towards.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+### Prerequisites
+
+- [Kubernetes Basics Tutorial](/docs/tutorials/kubernetes-basics/)
+
+{% endcapture %}
+
+{% capture objectives %}
+
+- Explore the different approaches for managing applications supported by *kubectl*.
+- Explore which environment each approach is best suited for.
+
+{% endcapture %}
+
+
+{% capture lessoncontent %}
+
+# App Management Styles Table
+
+**Currently, a given resource should be managed using only 1 style.  Attempting
+to mix-and-match styles for the same resource will result in undefined behavior.**
```

Comment:
Upon further reflection, I feel like this should live in specific docs for each technique.  Deleted this section for now and will add it to those docs when they are written.